funcx_sdk/funcx/sdk/executor.py

```diff
@@ -262,10 +262,10 @@ def _submit_task_kernel(self, kill_event: threading.Event):
             except queue.Empty:
                 pass
             if tasks:
-                log.info(f"Submitting tasks to funcX: {len(tasks)}")
+                log.debug(f"Submitting tasks to funcX: {len(tasks)}")
```

Comment:
This might be nice to stay as an info - this is the kind of log I'd want to see when watching logs in real time, to be sure that things are happening as expected. Same class of information as the result processor's `result received` or the web service logging its requests, both of which go to `info`.

Reply:
I agree with Chris on this. The user would see these log lines, only if they enable logging and I think the logline is important enough to stay at `info` level.

Reply:
Alright.  "Unchanged."